### PR TITLE
Reserve "juju" and "juju-*" actions

### PR DESCRIPTION
**Pull request by axw**
Just as we do for relation names, reserve the
action names "juju", and anything starting with
"juju-".

### CODE REVIEW

File: actions.go [hunk 1]
```diff
@@ -109,6 +109,9 @@ func ReadActionsYaml(r io.Reader) (*Actions, error) {
 		if valid := actionNameRule.MatchString(name); !valid {
 			return nil, fmt.Errorf("bad action name %s", name)
 		}
+		if reservedName(name) {
+			return nil, fmt.Errorf("reserved action name %s", name)
```
Comment (alesstimec) [inline actions.go]: i'd rather see a more useful message.. perhaps: `action name may not start with a reserved prefix "juju"`

**Comment by alesstimec** [comment]
LGTM with a nitpick

File: actions.go (hunk 1 above)
Comment (axw) [inline actions.go]: I've formatted/worded it a little differently, but kept with the spirit of making the error message human-readable.

**Comment by axw** [comment]
$$merge$$

**Pull request merged**